package-lock.json

```diff
@@ -0,0 +1,96 @@
+{
```

Comment:
Remove this file. We're using `yarn` not `npm`, so we the only lockfile we need is `yarn.lock`.